src/core/workspace/folder-to-project.ts

```diff
@@ -8,6 +8,9 @@ import { formatResourceType, formatResourceKey, formatAsHexString } from "@s4tk/
 import { findGlobMatches, parseKeyFromTgi } from "#building/resources";
 import StringTableJson from "#stbls/stbl-json";
 import * as inference from "#indexing/inference";
+var sanitize = require("sanitize-filename");
+
+var instanceMap: Map<bigint, [string, string]>;
```

Comment:
Since this is just used in the functions and re-assigned each time, create it in the `createFolderToProject()` function, and pass it as an argument to `_processResource()`.